app/assets/stylesheets/docs.scss

```diff
@@ -0,0 +1,117 @@
+$blue: #53ADC6;
+$red: #CF726E;
+$green: #5BC89E;
+$white: #f0f0f0;
+
+html,
+body {
+  font-family: 'Lato', serif;
+  font-size: 18px;
+  line-height: 1.62em;
+  margin: 0;
+}
+
+img {
+  max-width: 100%;
+}
+
+h1,
+h2,
+h3 {
+  font-family: 'Fjalla One', sans-serif;
+  margin-top: 2em;
+}
+
+code {
+  font-size: .8rem;
+  padding: .4em .2em 0;
```

Comment:
`.4` should be written with a leading zero as `0.4`<br>`.2` should be written with a leading zero as `0.2`